Xamarin.Forms.Platform.Android/Renderers/ShellToolbarTracker.cs

```diff
@@ -372,6 +386,7 @@ protected virtual void UpdateNavBarVisible(Toolbar toolbar, Page page)
 		protected virtual void UpdatePageTitle(Toolbar toolbar, Page page)
 		{
 			_toolbar.Title = page.Title;
+			_defaultNavigationContentDescription = _toolbar.SetNavigationContentDescription(page, _defaultNavigationContentDescription);
```

Comment:
Is this still right? since we determined NavigationContentDescription is for the icon?